### PR TITLE
[tinyexr] update version and add targets

### PR DESCRIPTION
**Pull request by fran6co**
This is part of https://github.com/microsoft/vcpkg/pull/12199

### CODE REVIEW

File: ports/tinyexr/portfile.cmake [hunk 1]
```diff
@@ -1,12 +1,21 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO syoyo/tinyexr
-    REF fdaeecbbc1bac1a417c6af2f4dab43d3840ed036
-    SHA512 0d875e197bc82dd5addb8e08e9ba1a9a02e108cc9dd14e5ea17de032cda5ad599f6d10d6c7959d545e05ab30d6daa34f98061f23faeb1c15caf96b6cb76483a3
+    REF 58a81c36caad469aed86441cc91080f23b496ffb
```
Comment (NancyLi1013) [inline ports/tinyexr/portfile.cmake:6]: This seems not the latest commit, why do you not use the latest one?

File: ports/tinyexr/CONTROL [hunk 2]
```diff
@@ -1,4 +1,4 @@
 Source: tinyexr
-Version: 2020-03-15
+Version: 2020-07-17
```
Comment (NancyLi1013) [inline ports/tinyexr/CONTROL:2]: How do you get this version?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/tinyexr/portfile.cmake (hunk 1 above)
Comment (fran6co) [reply ports/tinyexr/portfile.cmake:6]: I got it from the parent PR for open3d, I'll try to upgrade to latest and see filament compiles with it

File: ports/tinyexr/CONTROL (hunk 2 above)
Comment (fran6co) [reply ports/tinyexr/CONTROL]: switched to 1.0.0 now

**Review by NancyLi1013 — APPROVED** [review]

**Review by ras0219-msft — APPROVED** [review]
Thanks for the PR!

**Comment by BillyONeal** [comment]
Thanks again!

**Pull request merged**